ctapipe_io_lst/containers.py

```diff
@@ -22,185 +16,6 @@
-        [],
-        'Range of time of the pedestal events [t_min, t_max]',
-        unit=u.s
-    )
-    charge_mean = Field(
-        None,
-        "np array of pedestal average (n_chan, n_pix)"
-    )
-    charge_median = Field(
-        None,
-        "np array of the pedestal  median (n_chan, n_pix)"
-    )
-    charge_std = Field(
-        None,
-        "np array of the pedestal standard deviation (n_chan, n_pix)"
-    )
-    charge_median_outliers = Field(
-        None,
-        "Boolean np array of the pedestal median outliers (n_chan, n_pix)"
-    )
-    charge_std_outliers = Field(
-        None,
-        "Boolean np array of the pedestal std outliers (n_chan, n_pix)"
-    )
-
-
-class PixelStatusContainer(Container):
-    """
-    Container for pixel status information
-    It contains masks obtained by several data analysis steps
-    At r0 level only the hardware_mask is initialized
-    """
-    hardware_mask = Field(
-        None,
-        "Boolean np array (mask) from the hardware pixel status data (n_chan, n_pix)"
-    )
-
-    pedestal_mask = Field(
-        None,
-        "Boolean np array (mask) from the pedestal data analysis (n_chan, n_pix)"
-    )
-
-    flatfield_mask = Field(
-        None,
-        "Boolean np array (mask) from the flat-flield data analysis (n_chan, n_pix)"
-    )
-
-
-class WaveformCalibrationContainer(Container):
-    """
-    Container for the pixel calibration coefficients
-    """
-    time = Field(0, 'Time associated to the calibration event', unit=u.s)
-    time_range = Field(
-        [],
-        'Range of time of validity for the calibration event [t_min, t_max]',
-        unit=u.s
-    )
-
-    dc_to_phe = Field(
-        None,
-        "np array of (digital count) to (photon electron) coefficients (n_chan, n_pix)"
-    )
-
-    time_correction = Field(
-        None,
-        "np array of time correction values (n_chan, n_pix)"
-    )
-
-    n_phe = Field(
-        None,
-        "np array of photo-electrons in calibration signal (n_chan, n_pix)"
-    )
-
-    pixel_status_mask = Field(
-        None,
-        "Boolean np array (mask) of final calibration data analysis (n_chan, n_pix)"
-    )
-
-class MonitoringCameraContainer(Container):
-    """
-    Container for camera monitoring data
-    """
-
-    flatfield = Field(FlatFieldContainer(), "Data from flat-field event distributions")
-    pedestal = Field(PedestalContainer(), "Data from pedestal event distributions")
-    pixel_status = Field(PixelStatusContainer(), "Container for masks with pixel status")
-    calibration = Field(WaveformCalibrationContainer(), "Container for calibration coefficients")
-
-
-class MonitoringContainer(Container):
-    """
-    Root container for monitoring data (MON)
-    """
-
-    tels_with_data = Field([], "list of telescopes with data")
-
-    # create the camera container
-    tel = Field(
-        Map(MonitoringCameraContainer),
-        "map of tel_id to MonitoringCameraContainer")
-
```

Comment:
Just to keep track of things, all these containers are the ones that have been moved to `ctapipe/calib/camera`

Reply:
yes, they are all in ctapipe now